src/Ombi.Core/Engine/MovieRequestEngine.cs

```diff
@@ -533,7 +533,10 @@ private async Task CheckForSubscription(string UserId, List<MovieRequests> movie
                 }
                 else
                 {
-                    x.ShowSubscribe = true;
+                    if (!x.Available && !x.Available4K && (!x.Denied ?? true) && (!x.Denied4K ?? true))
```

Comment:
If 4k feature is not enabled, this will never be true as `Available4K` will always be false.

Probably want to do something like this?

```
if ((!x.Available || !x.Available4K) && ((!x.Denied ?? true) || (!x.Denied4K ?? true)))
```

Reply:
> If 4k feature is not enabled, this will never be true as `Available4K` will always be false.

Yes so !x.Available4K will be true and all is fine, no? I've found it to be quite accurate on a non 4k-enabled setups during my tests. ShowSubscribe will be true when the movie is not available at all, false if it's available in any quality. 

However now that you point it out, it may not behave as expected if a 4K request and a regular one is made and only one of those is closed. I'll look into it.